upload/views/test_results.py

```diff
@@ -59,7 +58,6 @@ class TestResultsView(
 ):
     permission_classes = [UploadTestResultsPermission]
     authentication_classes = [
-        UploadTokenRequiredAuthenticationCheck,
```

Comment:
TR needs a custom method, this is in an upcoming pr. We can leave this auth method in the stack, but it would never evaluate to True.